modules/network-observability-create-network-policy.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/network-observability-network-policy.adoc
+
+
+:_content-type: PROCEDURE
+[id="network-observability-network-policy_{context}"]
+= Create a network policy for Network Observability
+You might need to create a network policy to secure ingress traffic to the 'netobserv' namespace. In the web console, you can create a network policy using the form view.
+
+.Procedure
+. Navigate to *Networking* -> *NetworkPolicies*.
+. Select the `netobserv` project from the *Project* dropdown menu.
+. Name the policy. For this example, the policy name is `allow-ingress`.
+. Click *Add ingress rule* three times to create three ingress rules.
```

Comment:
I always get confused on this. Is "Ingress" capitalized? https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress https://redhat-documentation.github.io/supplementary-style-guide/#bare-metal-n aren't 100% clear to me.

Reply:
I think that if you're referencing a resource, like the Ingress Controller, then you capitalize, but if you're referencing traffic, like ingress or egress traffic, then its not capitalized. In this case I'm talking about traffic, so I think its lowercase and its also lowercase in the web console form that I'm referencing.